### ISSUE TITLE
How do I set the options on map creation?

### ISSUE THREAD

**Issue by fccoelho** [leplatrem/django-leaflet-geojson#5]
More documentation is needed to help customize the map.

**Comment by leplatrem** [comment]
Yes, indeed, but more documentation is available on [django-leaflet project page](https://github.com/makinacorpus/django-leaflet) though.

The map object will be already initialized, but you could still do this :

``` javascript
   // as stated in django-leaflet readme
   window.addEventListener("map:init", function (e) {
        var map = e.detail.map;
        map.setOptions( {...} );
        ...
    }, false);

```

Tell me if you encounter problems setting options after map initialization. Maybe it's too late for some options to be taken into account. In this case, please open an issue (with the name of the options) in django-leaflet directly :)

Thanks ! Good luck !
